appveyor.yml

```diff
@@ -51,34 +37,11 @@ before_build:
   - SET PATH=%PYTHON%;%PYTHON%\Scripts;%PATH%
   - python --version
   - pip install -r src/utils/cbse/requirements.txt --upgrade
-  - ps: |
-      if ($env:COMPILER -eq "mingw64" -AND -Not (Test-Path "C:/mingw64")) {
-          $url  = "http://sourceforge.net/projects/mingw-w64/files/"
-          $url += "Toolchains%20targetting%20Win64/Personal%20Builds/"
-          $url += "mingw-builds/${env:COMPILER_VERSION}/threads-win32/seh/"
-          $url += "x86_64-${env:COMPILER_VERSION}-release-win32-seh-rt_${env:MINGW_VERSION}.7z/download"
-          Invoke-WebRequest -UserAgent wget -Uri $url -OutFile mingw64.7z
-          &7z x -oC:\ mingw64.7z > $null
-          $env:PATH = "C:/mingw64/bin;" + ($env:PATH -replace [regex]::escape("C:\Program Files (x86)\Git\bin"), "")
-      }
 
 build_script:
   - cmd: cmake --version
   - cmd: mkdir build
   - cmd: cd build
-  - ps: |
-      if ($env:COMPILER -eq "msvc") {
-          pushd "${env:ProgramFiles(x86)}/Microsoft Visual Studio ${env:COMPILER_VERSION}/VC/"
-          cmd /c "vcvarsall.bat $env:TARGET & set" |
-          foreach {
-              if ($_ -match "=") {
-                  $v = $_.split("="); set-item -force -path "ENV:\$($v[0])"  -value "$($v[1])"
-              }
-          }
-          popd
-      } elseif ($env:COMPILER -eq "mingw64") {
-          $env:CC="gcc"
-          $env:CXX="g++"
-      }
-      cmake -DUSE_PRECOMPILED_HEADER=0 -DUSE_WERROR=1 -DBE_VERBOSE=1 -G "${env:GENERATOR}" -DBUILD_GAME_NACL=0 -DCMAKE_BUILD_TYPE=Debug ..
-      cmake --build .
+  - cmd: call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvars32.bat"
+  - cmd: cmake -DUSE_PRECOMPILED_HEADER=0 -DUSE_WERROR=1 -DBE_VERBOSE=1 -G "NMake Makefiles" -DBUILD_GAME_NACL=0 -DCMAKE_BUILD_TYPE=Debug ..
```

Comment:
Since we build the client, tty-client and server in daemon already; is it still useful to build it here, too?

Reply:
yeah, good point, can be seen as a leftover from before repo split to my point of view